src/markdown.cpp

```diff
@@ -352,6 +352,37 @@ static Alignment markersToAlignment(bool leftMarker,bool rightMarker)
   }
 }
 
+/** parse the image attributes and return attributes for given format */
+static QCString getImageAttributes(const char *fmt, const StringVector &attrList)
+{
+  QCString retval;
+  for (const auto &attr_ : attrList)
+  {
+    QCString attr = QCString(attr_).stripWhiteSpace().lower();
+    int i = attr.find(':');
+    if (i>0)
+    {
+      // multiple formats
+      QCString format = attr.left(i).stripWhiteSpace();
+      if (!format.isEmpty())
+      {
+        if (format == fmt)
+        {
+          retval = attr.right(attr.length() - i - 1).data();
+          break;
+        }
```

Comment:
Wouldn't this prevent something like:
```
...{html: width=5cm, html: height=50%}
```

Reply:
should we consider such situation or just document first item is taken?

Reply:
I think we should consider this situation.
I think that it should be possible to have e.g.
```
{html: width=5cm, html: height=50%, latex: width=4cm, latex: height=40%}
```
I don't think something like:
```
{html: width=5cm, height=50%, latex: width=4cm, height=40%}

```
should be supported (is more compact but will be harder to parse and would probably be less flexible).

Reply:
At this moment the format would be 

```
![caption](image.png){width=50% height=10%}
![caption](image.png){html: width=100px height=250px, latex: width=5cm height=10cm}
```

So I consume all attributes after the first valid format definition and pass them on to the image command which is defined and used as:

```
\image['{'option[,option]'}'] <format> <file> ["caption"] [<sizeindication>=<size>]
\image html image.png width=100px height=250px
\image latex image.png width=5cm height=10cm
```

I'm not sure adding format definition for each sizeindication is making things clearer but can be done...